### PR TITLE
Converting all variable type definitions from var to const or let

### PR DESCRIPTION
**Pull request by pacostas**
Replacing var type definitions in documentation to const or let.

### CODE REVIEW

**Review by abstractj — DISMISSED** [review]

**Comment by abstractj** [comment]
@jonkoops LGTM, but I will wait for your approval before merge.

File: aggregation/navbar.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.js"></script>
 
 <script>
-    var oldtoc = document.getElementById('toctitle').nextElementSibling;
-    var newtoc = document.createElement('div');
+    const oldtoc = document.getElementById('toctitle').nextElementSibling;
+    let newtoc = document.createElement('div');
```
Comment (jonkoops) [inline aggregation/navbar.html]: Why is `newtoc` defined with `let`? As far as I can see this variable is never re-assigned so should be fined to make it a constant.
```suggestion
    const newtoc = document.createElement('div');
```

File: authorization_services/topics/enforcer-js-adapter.adoc [hunk 2]
```diff
@@ -46,7 +46,7 @@ and use the library to send an authorization request as follows:
 
 ```javascript
 // prepare a authorization request with the permission ticket
-var authorizationRequest = {};
+let authorizationRequest = {};
```
Comment (jonkoops) [inline authorization_services/topics/enforcer-js-adapter.adoc]: Same here, this is never re-assigned so can also be constant.
```suggestion
const authorizationRequest = {};
```

File: authorization_services/topics/policy-js-policy.adoc [hunk 3]
```diff
@@ -134,7 +134,7 @@ To push arbitrary claims to the resource server in order to provide additional i
 enforced:
 
 ```javascript
-var permission = $evaluation.getPermission();
+let permission = $evaluation.getPermission();
```
Comment (jonkoops) [inline authorization_services/topics/policy-js-policy.adoc]:
```suggestion
const permission = $evaluation.getPermission();
```

File: securing_apps/topics/oidc/javascript-adapter.adoc [hunk 4]
```diff
@@ -116,12 +116,12 @@ After the user is authenticated the application can make requests to RESTful ser
...
 ----
-var loadData = function () {
+const loadData = function () {
     document.getElementById('username').innerText = keycloak.subject;
 
-    var url = 'http://localhost:8080/restful-service';
+    const url = 'http://localhost:8080/restful-service';
 
-    var req = new XMLHttpRequest();
+    let req = new XMLHttpRequest();
```
Comment (jonkoops) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
    const req = new XMLHttpRequest();
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
Looks like we have some failing tests. Can we get a rebase?

**Comment by pacostas** [comment]
Sure, just rebased it

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
The problems with external links are unrelated to the changes here. @abstractj could you go ahead and merge this PR?

**Review by abstractj — APPROVED** [review]

**Comment by jonkoops** [comment]
@pacostas Thanks for your contribution :100:

**Pull request merged**